__tests__/index.test.js

```diff
@@ -158,6 +158,50 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST: 201 adds a new comment to a particular article", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "NC is the best software development bootcamp",
+    };
+
+    return request(app)
+      .post("/api/articles/11/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment.author).toBe("butter_bridge");
+        expect(comment.body).toBe(
+          "NC is the best software development bootcamp"
+        );
+      });
```

Comment:
Probably better to compare with the whole comment with all the properties, especially an article_id: expect.any(number)